src/profile-v2/ProfilePage.jsx

```diff
@@ -42,6 +48,8 @@ const ProfilePage = ({ params }) => {
   } = useSelector(profilePageSelector);
 
   const [viewMyRecordsUrl, setViewMyRecordsUrl] = useState(null);
+  const isMobileView = useWindowSize().width <= breakpoints.small.minWidth;
+  const isTabletView = useWindowSize().width <= breakpoints.medium.minWidth;
```

Comment:
Create generic custom hooks as shown below and reuse them wherever needed:
export function useIsOnTabletScreen() {
  const windowSize = useWindowSize();
  return windowSize.width <= breakpoints.medium.minWidth;
}

export function useIsOnMobileScreen() {
  const windowSize = useWindowSize();
  return windowSize.width <= breakpoints.small.minWidth;
}